_topic_map.yml

```diff
@@ -331,7 +331,38 @@ Topics:
 - Name: Disabling the web console
   File: disabling-web-console
   Distros: openshift-enterprise,openshift-webscale,openshift-origin
-
+---
+Name: Security
```

Comment:
Sounds good.

Reply:
I put down some thoughts in an email. Please add your own thoughts and feel free to send it to the folks you noted above.

Reply:
Since everyone seems in agreement, I'm going to assume we are heading toward a top-level "Security" bucket for holding this content. So I'll resolve this issue for now.

Reply:
Looks like we will have the new Security bucket, so I'll resolve this.